HW_6/HW_7.py

```diff
@@ -0,0 +1,248 @@
+import datetime
+import random
+import datefinder
+import HW_3_updated as normalization
+import sys, os
+import csv
+import re
+from collections import Counter
+import json
+import xml.etree.ElementTree as ET
```

Comment:
The boring recommendation about imports in PEP8 :)

Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Imports should be grouped in the following order:

1. Standard library imports (os/sys/re etc).
2. Related third party imports (pyinputplus).
3. Local application/library specific imports.

You should put a blank line between each group of imports. Also, all libraries within group should be sorted alphabetically (asc)